### DISCUSSION TITLE
tagによる検索でヒットしないのは何故

### DISCUSSION THREAD

**Discussion by nak435** [increments/qiita-discussions#534] [要望・アイデア・不具合]
今朝、`tag: CH32V203`の条件で記事を検索した時、以前、自分が投稿した記事が1件少ないことに気づき、
タグを付け忘れたのかと思ったのですが、タグ自身をクリックすると、出てきました。
タグによる検索でヒットしなかったのはなぜでしょうか？

2番目のスクショの先頭の記事が、タグによる検索でヒットしなかった記事です。

![search_for_tag](https://github.com/increments/qiita-discussions/assets/42625437/b04d5cd1-91e0-4ae6-b816-2eed5038e4b7)

<hr>

![click_tag](https://github.com/increments/qiita-discussions/assets/42625437/53a8244b-62e4-4f63-8316-237683bb95c9)

**Comment by tanaka-asami415** [comment]
@nak435 
ご報告ありがとうございます。

ご不便をおかけしてしまい、申し訳ございません。
バグと思われるため調査して対応します！🙇

**Reply by nak435** [reply]
同条件で検索すると、件数が一致していました。
いつの間にか、対策されたようです。

**Reply by tanaka-asami415** [reply]
@nak435 
ご報告が遅れて申し訳ありません🙇

原因はわかっていないのですが、特定の状況で検索システムに反映がうまくされない様でした。
定期的に検索システムに再反映するようにしたので、現在は解消されています。
今後同症状が現れる場合は、また教えていただけると幸いです。

**Reply by nak435** [reply]
ありがとうございます。本件はクローズいたします。
検索キーワードに対する追加要望を別に起票します。
